README.md

```diff
@@ -1,18 +1,26 @@
 # 1bit-ui
 
-See [1bit-ui.com](https://1bit-ui.com) for documentation and usage.
+1bit-ui is a CSS library for making retro-inspired web pages using just two colors.
+
+Go to [1bit-ui.com](https://1bit-ui.com) for examples, documentation, and usage.
 
 ## Development
 
+You can use npm to load 1bit-ui on a local server.
+
 ```
 $ npm install
 $ npm start
 ```
 
 Edit `src/1bit-ui.css`. It's the only file that gets published to npm. Open your browser to <http://localhost:5000/>.
 
+> If you're unfamiliar with npm, you can [download the npm installer here](https://www.npmjs.com/get-npm) and read more about [how to use npm here](https://docs.npmjs.com/about-npm/).
```

Comment:
npm comes preinstalled with node, so i would just link to the node installer

Reply:
I made a couple slight changes :)